src/shared/logger/activation.ts

```diff
@@ -75,8 +77,10 @@ export async function activate(
 
     await registerLoggerCommands(extensionContext)
     logOutputChannel.appendLine(
-        localize('AWS.log.fileLocation', 'Error logs for this session are permanently stored in {0}', logPath)
+        localize('AWS.log.fileLocation', 'Error logs for this session are stored in {0}', logPath)
```

Comment:
the entry in `package.nls.json` can be deleted.

while we're here, let's make this message less verbose and more accurate (not all logs are "error" logs):
```suggestion
        localize('AWS.log.fileLocation', 'Log file: {0}', logPath)
```